docs/roadmap.md

```diff
@@ -9,39 +9,16 @@ This document highlights some ideas for major features we'd like to implement in
+Cortex was accepted to the CNCF on September 20, 2018 and moved to the Incubating maturity level on August 20, 2020. The Cortex maintainers are working towards promoting the project to the graduation status. See [issue #6075](https://github.com/cortexproject/cortex/issues/6075) for tracking this progress.
 
-## Auth Gateway
+## Support for Prometheus Remote Write 2.0
 
-Cortex server has a simple authentication mechanism (X-Scope-OrgId) but users can't use the multitenancy features out of the box without complicated proxy configuration. It's hard to support all the different authentication mechanisms used by different companies but plan to have a simple but opinionated auth-gateway that provides value out of the box. The configuration could be as simple as:
+[Prometheus Remote Write 2.0](https://prometheus.io/docs/specs/remote_write_spec_2_0/)
 
-```
-tenants:
-- name: infra-team
-  password: basic-auth-password
-- name: api-team
-  password: basic-auth-password2
-```
+* adds a new Protobuf Message with new features enabling more use cases and wider adoption on top of performance and cost savings
+* deprecates the previous Protobuf Message from a 1.0 Remote-Write specification
+* adds mandatory X-Prometheus-Remote-Write-*-Written HTTP response headers for reliability purposes
 
-## Billing and Usage analytics
-
-We have all the metrics to track how many series, samples and queries each tenant is sending but don't have dashboards that help with this. We plan to have dashboards and UIs that will help operators monitor and control each tenants usage out of the box.
-
-## Downsampling
-Downsampling means storing fewer samples, e.g. one per minute instead of one every 15 seconds.
-This makes queries over long periods more efficient. It can reduce storage space slightly if the full-detail data is discarded.
-
-## Per-metric retention
-
-Cortex blocks storage supports deleting all data for a tenant after a time period (e.g. 3 months, 1 year), but we would also like to have custom retention for subsets of metrics (e.g. delete server metrics but retain business metrics).
-
-## Exemplar support
-[Exemplars](https://docs.google.com/document/d/1ymZlc9yuTj8GvZyKz1r3KDRrhaOjZ1W1qZVW_5Gj7gA/edit)
-let you link metric samples to other data, such as distributed tracing.
-As of early 2021 Prometheus will collect exemplars and send them via remote write, but Cortex needs to be extended to handle them.
-
-## Scalability
-
-Scalability has always been a focus for the project, but there is a lot more work to be done. We can now scale to 100s of Millions of active series but 1 Billion active series is still an unknown.
```

Comment:
1 Billion active series in single tenant is doable today. 
There are of course other scalability points we could still implement. I don't think we should put them in the roadmap, they are not that critical